LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/animations/font-variation-settings-interpolation-expected.txt

```diff
@@ -353,8 +353,8 @@ PASS Web Animations: property <font-variation-settings> from ['aaaa' 0, 'bbbb' 1
-FAIL Web Animations: property <font-variation-settings> from ['aaaa' 30, 'bbbb' 20] to ['aaaa' 20, 'bbbb' 30] at (3.40282e+38) should be ['aaaa' -3.40282e+38, 'bbbb' 3.40282e+38] assert_array_equals: expected property 0 to be "\"aaaa\" -340282001837565597733306976381245063168" but got "\"aaaa\" -infinity" (expected array ["\"aaaa\" -340282001837565597733306976381245063168", "\"bbbb\" 340282001837565597733306976381245063168"] got ["\"aaaa\" -infinity", "\"bbbb\" infinity"])
+FAIL CSS Transitions: property <font-variation-settings> from ['aaaa' 30, 'bbbb' 20] to ['aaaa' 20, 'bbbb' 30] at (3.40282e+38) should be ['aaaa' -3.40282e+38, 'bbbb' 3.40282e+38] assert_array_equals: expected property 0 to be "\"aaaa\" -340282001837565597733306976381245063168" but got "\"aaaa\" -340282346638528859811704183484516925440" (expected array ["\"aaaa\" -340282001837565597733306976381245063168", "\"bbbb\" 340282001837565597733306976381245063168"] got ["\"aaaa\" -340282346638528859811704183484516925440", "\"bbbb\" 340282346638528859811704183484516925440"])
+FAIL CSS Transitions with transition: all: property <font-variation-settings> from ['aaaa' 30, 'bbbb' 20] to ['aaaa' 20, 'bbbb' 30] at (3.40282e+38) should be ['aaaa' -3.40282e+38, 'bbbb' 3.40282e+38] assert_array_equals: expected property 0 to be "\"aaaa\" -340282001837565597733306976381245063168" but got "\"aaaa\" -340282346638528859811704183484516925440" (expected array ["\"aaaa\" -340282001837565597733306976381245063168", "\"bbbb\" 340282001837565597733306976381245063168"] got ["\"aaaa\" -340282346638528859811704183484516925440", "\"bbbb\" 340282346638528859811704183484516925440"])
+FAIL CSS Animations: property <font-variation-settings> from ['aaaa' 30, 'bbbb' 20] to ['aaaa' 20, 'bbbb' 30] at (3.40282e+38) should be ['aaaa' -3.40282e+38, 'bbbb' 3.40282e+38] assert_array_equals: expected property 0 to be "\"aaaa\" -340282001837565597733306976381245063168" but got "\"aaaa\" -340282346638528859811704183484516925440" (expected array ["\"aaaa\" -340282001837565597733306976381245063168", "\"bbbb\" 340282001837565597733306976381245063168"] got ["\"aaaa\" -340282346638528859811704183484516925440", "\"bbbb\" 340282346638528859811704183484516925440"])
+FAIL Web Animations: property <font-variation-settings> from ['aaaa' 30, 'bbbb' 20] to ['aaaa' 20, 'bbbb' 30] at (3.40282e+38) should be ['aaaa' -3.40282e+38, 'bbbb' 3.40282e+38] assert_array_equals: expected property 0 to be "\"aaaa\" -340282001837565597733306976381245063168" but got "\"aaaa\" -340282346638528859811704183484516925440" (expected array ["\"aaaa\" -340282001837565597733306976381245063168", "\"bbbb\" 340282001837565597733306976381245063168"] got ["\"aaaa\" -340282346638528859811704183484516925440", "\"bbbb\" 340282346638528859811704183484516925440"])
```

Comment:
Provocative failing results; I wonder what it would take to make this test pass.